### DISCUSSION TITLE
Elevage d'animaux

### DISCUSSION THREAD

**Discussion by niqore** [Crownicles/Crownicles#621] [Nouvelles idées]
Pouvoir accoupler des animaux (à la base ça devait servir à ça les genres) mais attention à ne pas le rendre trop OP. Pour ça je propose de rendre l'élevage de plus en plus compliqué avec la rareté de l'animal.
Aussi pourquoi pas donner des âges aux animaux : ils naissent bébés et ils grandissent. Après je pense que ça va être un peu chiant mais on pourrait imaginer qu'ils vieillissent et qu'ils meurent pour ne pas rendre les pets trop communs non plus

**Comment by Chevalier2** [comment]
Combien de temps un pet mettrait à mourir ? Il faudrait laisser pas mal de temps je pense. Ensuite, les bébés seront-ils directement dressés ? C'est ce qu'il faudrait faire je pense. Aussi, pourquoi pas une durée de vie en fonction de la rareté ( ou même du pet ) ?

**Comment by Pietagorh** [comment]
ça serait chiant je trouve, pas besoin de surcompliquer les pets
